cves/kernel/CVE-2012-2375.yml

```diff
@@ -251,8 +275,10 @@ i18n:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: >-
+    Was just an issue with how long the length of the variable is. It did not
+    mention anything related to internationalization.
```

Comment:
Maybe define some markers of internationalization for clarification.

Reply:
Updated